meinberlin/assets/scss/_form.scss

```diff
@@ -26,6 +35,16 @@ select {
     }
 }
 
+// FIXME ensure auto added buttons look inline with rest of UI eg:close on alerts
+button {
+    appearance: none;
+    color: inherit;
+    background: none;
+    font-size: inherit;
+    border: 0;
+    cursor: pointer;
+}
+
```

Comment:
I am not sure if this should be needed for any of the BO elements as this and control base should already be set in bo styles?!

Reply:
It wasn't set in bo styles. I took this `button` style over because without it buttons look like native buttons with that grey background and border but our default style is "make them not look like a button". 

regarding the control-base, I can remove the entire mixin and where it was used, i.e. [here](https://github.com/liqd/a4-meinberlin/pull/5322/files#diff-8386b6a134ac758dcc457ca8b8c73525c88299d1ad799353ae2ee39ed2bb0963R13)?

Reply:
Ok I need to check their stuff again as I thought they set their bases, I was going to restructure the frontend files and only have the old components, including this, loaded for the dashboard and only load BO and our additional style sheets for user side pages but if they haven't set up base styles that won't work, I also wonder how their styleguide is working then?!

Reply:
They only remove the base style for specific variants: https://styleguide.berlin.de/patterns/09-vertical_marketing-page-element-buttons/09-vertical_marketing-page-element-buttons.html look at the clean button or the close button